src/main/scala/cromwell/server/WorkflowManagerSystem.scala

```diff
@@ -7,7 +7,6 @@ import cromwell.engine.workflow.WorkflowManagerActor
 trait WorkflowManagerSystem {
   val systemName = "cromwell-system"
   implicit val actorSystem = ActorSystem(systemName)
+  lazy val dataAccess: DataAccess = DataAccess()
```

Comment:
Tho might not be a bad idea to add a comment to that effect. I wouldn't say doing so was necessary though, but I'll probably come across and wonder "Why lazy?" again a week from now :)